__app__/sitemapparser/sites/sumosear_ch.py

```diff
@@ -0,0 +1,27 @@
+from urllib.parse import urljoin
+from bs4 import BeautifulSoup
+
+#https://sumosear.ch/images/tags/carbondale-il/escorts
+#https://sumosear.ch/images/tags/carbondale-il/female-escorts
+
+tags = [
+    "escorts",
+    "female-escorts",
+    "massage-body-rubs",
+    "male-escorts",
+    "trans-shemale-escorts",
+    "fetish-domination"
+]
+
+def sumosear_ch(html):
+    soup = BeautifulSoup(html, "html.parser")
+    links = []
+    locations = soup.find("a", class_="location-bar__location-btn location-btn js-modal-opener").get("data-full-tags")
+    locations = eval(locations)
+    location_tags = []
+    for location in locations:
+        location_tags.append(location[1])
+    for location_tag in location_tags:
+        for tag in tags:
+            links.append(f"https://sumosear.ch/images/tags/{location_tag}/{tag}")
+    return links
```

Comment:
```suggestion
    locations = soup.find("a", class_="location-bar__location-btn location-btn js-modal-opener").get("data-full-tags")
    locations = json.loads(locations)
    for _, location_tag in locations:
        for tag in tags:
            links.append(f"https://sumosear.ch/images/tags/{location_tag}/{tag}")
    return links
    
```
The main thing here is not to use eval. At least in ruby it is a security thing.
I switched it up to use json.

You'll need to import it at the top.